packages/l10n/data/currencies/de.json

```diff
@@ -181,7 +181,7 @@
   },
   "DZD": {
     "label": "Algerischer Dinar",
-    "symbol": "د.ج.‏"
+    "symbol": "DA"
```

Comment:
Hmm why did those change? Did you install the latest deps?

Reply:
I thought I did!
Updated the deps, and re-ran the `generate-data`